fission/environment.go

```diff
@@ -41,6 +41,13 @@ func envCreate(c *cli.Context) error {
 		fatal("Need an image, use --image.")
 	}
 
+	envBuilderImg := c.String("builder")
+
+	envBuildCmd := c.String("buildcmd")
```

Comment:
Let's use a default value so users can avoid typing one more option on the CLI.
How about simply `"build"`.

Reply:
so the environment builder need to provide a default `build` command to build the package?

Reply:
Yeah.